src/emqx.erl

```diff
@@ -234,7 +234,19 @@ shutdown(Reason) ->
                  ).
 
 reboot() ->
-    lists:foreach(fun application:start/1 , default_started_applications()).
+    case is_application_running(emqx_dashboard) of
+        true ->
+            application:stop(emqx_dashboard), %% dashboard must be started after mnesia
+            lists:foreach(fun application:start/1 , default_started_applications()),
+            application:start(emqx_dashboard);
+
+        false ->
+            lists:foreach(fun application:start/1 , default_started_applications())
+    end.
+
+is_application_running(App) ->
+    StartedApps = proplists:get_value(started, application:info()),
```

Comment:
@zmstone Do we have a package that does not enable `emqx_dashboard`?
I used to  think the `emqx_dashboard` is always enabled.

`application:info()` return `started, running, starting,start_p_false, loaded, loading` status.
I'm a bit curious if emqx_dashboard, which was started first without mnesia started, will cause it to stay in the `starting` state.

I would suggest removing `is_application_running(emqx_dashboard)` check if we always enable dashboard.

Reply:
Technically, it’s possible to disable dashboard.
It’s just as other plugins